modules/bliinkBidAdapter.js

```diff
@@ -57,51 +57,26 @@ export function getOneMetaValue(query) {
   return null
 }
 
-export function getMultipleMetaValue(query) {
-  const metaEls = document.querySelectorAll(query)
-
-  if (metaEls.length !== 0) {
-    const values = []
-
-    metaEls.forEach((metaEl) => {
-      if (metaEl.content) {
-        values.push(metaEl.content)
-      }
-    })
-
-    return values
-      .map((value) => value.trim())
-      .join(',')
-  }
-
-  return null
-}
-
 export function getMetaValue(name, type = 1) {
   const metaList = getMetaList(name)
-
-  let metaContent = ''
-
-  metaList.forEach((meta) => {
-    const method = type === 1 ? getOneMetaValue : getMultipleMetaValue
-    const metaValue = method(`meta[${meta.key}=${meta.value}]`)
+  const value = metaList.some((meta) => {
```

Comment:
`Array.some` [returns a boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some#return_value), not the value.

Probably best to just use a simple for-loop:
```
for (let i=0; i < metaList.length; i++) {
  const meta = metaList[i];
  const metaValue = getOneMetaValue(`meta[${meta.key}=${meta.value}]`);
  if (metaValue) {
    return metaValue
  }
}
return ''
```

Reply:
Thank @gwhigs for pointing  it out, we pushed the fix